roles/download/defaults/main.yml

```diff
@@ -221,6 +235,42 @@ downloads:
     repo: "{{ kibana_image_repo }}"
     tag: "{{ kibana_image_tag }}"
     sha256: "{{ kibana_digest_checksum|default(None) }}"
+  prometheus_operator:
+    container: true
+    repo: "{{ prometheus_operator_image_repo }}"
+    tag: "{{ prometheus_operator_image_tag }}"
+    sha256: "{{ prometheus_operator_digest_checksum|default(None) }}"
+  node_exporter:
+    container: true
+    repo: "{{ node_exporter_image_repo }}"
+    tag: "{{ node_exporter_image_tag }}"
+    sha256: "{{ node_exporter_digest_checksum|default(None) }}"
+  kube_state_metrics:
+    container: true
+    repo: "{{ kube_state_metrics_image_repo }}"
+    tag: "{{ kube_state_metrics_image_tag }}"
+    sha256: "{{ kube_state_metrics_digest_checksum|default(None) }}"
+  alertmanager:
```

Comment:
I don't see why all images are there. 
At somepoint we need to let kubernetes download the containers